src/main.ts

```diff
@@ -1,16 +1,26 @@
 import * as core from '@actions/core'
 import {action, error} from './config'
 import {getBody, contains, update, mutate} from './check'
-import {getPreviousBody, getCurrentBody, getDiff} from './detect'
+import {
+  getPreviousBody,
+  getCurrentBody,
+  getDiff,
+  getCurrentChecked,
+  getCurrentUnchecked
+} from './detect'
 
 async function run(): Promise<void> {
   try {
     if (action === 'detect') {
       const previousBody = getPreviousBody()
       const currentBody = getCurrentBody()
-      const {checked, unchecked} = getDiff(previousBody, currentBody)
+      const changed = getDiff(previousBody, currentBody)
+      const checked = getCurrentChecked(currentBody)
+      const unchecked = getCurrentUnchecked(currentBody)
+
       core.setOutput('checked', JSON.stringify(checked))
       core.setOutput('unchecked', JSON.stringify(unchecked))
+      core.setOutput('changed', JSON.stringify(changed))
```

Comment:
I dont want to make breaking change here.
Just make another action type for current.

also I would be happy if you update README.

Reply:
can you propose the output shape you'd like?

Reply:
The existing function returned a list of checks and unchecks for changes. I hope there is no change in that area. 
I think the ideal thing would be to do a current-detect action and return a list of checked and unchecked items.